jupyterlab_git/tests/test_clone.py

```diff
@@ -32,6 +32,41 @@ async def test_git_clone_success():
             assert {"code": 0, "message": output} == actual_response
 
 
+@pytest.mark.asyncio
+@pytest.fixture
+async def test_git_clone_success():
+    with patch("os.environ", {"TEST": "test"}):
+        with patch("jupyterlab_git.git.execute") as mock_execute:
+            # Given
+            output = "output"
+            mock_execute.return_value = maybe_future((0, output, "error"))
+
+            # Create a temporary directory and fake the content of it to be a directory containing a `.git` folder (that should be removed)
+            # I advice you to use the pytest fixture tmp_path; see https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-tmp_path
+            path = tmp_path_factory.mktemp("check-symlinks")
+            tmpdir = tempfile.mkdtemp()
+            temppath = tmp_path()
+
+            # When
+            actual_response = await Git().clone(
+                path=path,  # Here set the path to the temporary directory
```

Comment:
To pass the temporary folder to the clone method, you will probably need to convert it to a string

```suggestion
                path=str(temppath),  # Here set the path to the temporary directory
```